<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SweetDisplayStandby</class>
 <widget class="QMainWindow" name="SweetDisplayStandby">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>240</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>SweetDisplayStandby</string>
  </property>
  <property name="windowIcon">
   <iconset resource="sweetdisplaystandby.qrc">
    <normaloff>:/main/icon.png</normaloff>:/main/icon.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="brightnessGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>211</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Brightness preferences</string>
    </property>
    <widget class="QLabel" name="normalBrightnessLevelLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Brightness level in normal mode</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="normalBrightnessLevelInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="dimmedBrightnessLevelLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Brightness level in dimmed mode</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="dimmedBrightnessLevelInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="normalBrightnessLevelUnitsLabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>percents</string>
     </property>
    </widget>
    <widget class="QLabel" name="dimmedBrightnessLevelUnitsLabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>percents</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="timeoutGroupBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>211</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Timeout preferences</string>
    </property>
    <widget class="QLabel" name="turnOffTimeUnitsLabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>seconds</string>
     </property>
    </widget>
    <widget class="QLabel" name="dimTimeUnitsLabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>seconds</string>
     </property>
    </widget>
    <widget class="QLabel" name="turnOffTimeLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Turn off timeout</string>
     </property>
    </widget>
    <widget class="QLabel" name="dimTimeLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Dim timeout</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="turnOffTimeInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="dimTimeInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="brightnessStateCheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable brightness management</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="shortcutGroupBox">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>40</y>
      <width>211</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Shortcut preferences</string>
    </property>
    <widget class="QLabel" name="turnOffSequenceLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Display turn off shortcut</string>
     </property>
    </widget>
    <widget class="QKeySequenceEdit" name="turnOffSequenceEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QKeySequenceEdit" name="restoreSequenceEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="restoreSequenceLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Application restore shortcut</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>670</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionHideToTray"/>
    <addaction name="actionHideCompletely"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionProjectSite"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionHideToTray">
   <property name="text">
    <string>Hide to tray</string>
   </property>
  </action>
  <action name="actionHideCompletely">
   <property name="text">
    <string>Hide completely</string>
   </property>
  </action>
  <action name="actionProjectSite">
   <property name="text">
    <string>Project site</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sweetdisplaystandby.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>SweetDisplayStandby</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
